core/frontend/src/DisplayStyleState.ts

```diff
@@ -391,12 +394,23 @@ export abstract class DisplayStyleState extends ElementState implements DisplayS
   /** @internal */
   public getMapLayers(isOverlay: boolean) { return isOverlay ? this.overlayMapLayers : this.backgroundMapLayers; }
 
-  /** @internal */
-  public attachMapLayerSettings(settings: MapLayerSettings, isOverlay: boolean, insertIndex = -1): void {
+  /**
+   * Attach a map layer to display style.
+   * @param Settings representing the map layer.
+   * @param isOverlay true if layer is overlay, otherwise layer is background. Defaults to false.
+   * @param index where the layer should be inserted. Defaults to -1, appended to end.
+   * @public
+   *
+   */
+  public attachMapLayer(settings: MapLayerSettings, isOverlay?: boolean, insertIndex?: number): void {
```

Comment:
I thought we talked about using named parameters object instead of optional arguments here.

Reply:
We seem to use "options" more commonly than "args", but either works.